contracts/token_wrapper_handler/lib.rs

```diff
@@ -388,7 +388,7 @@ mod token_wrapper_handler {
             resource_id: [u8; 32],
             function_signature: [u8; 4],
             nonce: [u8; 8],
-            fee: [u8; 1],
+            fee: [u8; 2],
```

Comment:
Can we make this `u16` please and avoid byte arrays when we're dealing directly with numbers?